pkg/daemons/agent/agent.go

```diff
@@ -194,12 +194,27 @@ func checkCgroups() (root string, hasCFS bool, hasPIDs bool) {
 				if _, err := os.Stat(p); err == nil {
 					hasCFS = true
 				}
-			} else if system == "name=systemd" {
+			}
+		}
+	}
+
+	g, err := os.Open("/proc/1/cgroup")
```

Comment:
Can you add a comment explaining why we're checking the init process's cgroups as opposed to our own?